openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -196,11 +196,12 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         if permstore.user_oml == OrganizationManagementLevel.SUPERADMIN:
             return
 
+        self._check_for_higher_OML(permstore, instance)
         self._check_OML_in_instance(permstore, instance)
         actual_group_fields = self._get_actual_grouping_from_instance(instance)
 
         # Ordered by supposed velocity advantages. Changing order only can effect the sequence of detected errors for tests
-        self.check_group_E(permstore, actual_group_fields["E"])
+        self.check_group_E(permstore, actual_group_fields["E"], instance)
```

Comment:
Please revert the change to the params since it is no longer needed

Reply:
Done.